finat/morley.py

```diff
@@ -10,37 +10,42 @@ class Morley(PhysicallyMappedElement, ScalarFiatElement):
     def __init__(self, cell, degree=2):
         if Citations is not None:
             Citations().register("Morley1971")
-        super().__init__(FIAT.Morley(cell))
+        super().__init__(FIAT.Morley(cell, degree=degree))
 
     def basis_transformation(self, coordinate_mapping):
```

Comment:
I'm almost positive this wants to be written out on a white board and will need FIAT "tangent normals".  Since the tests fail, I'll assume your guess was wrong and come back to reviewing after I've done more math.
Also, I was wondering if we could just try this out as our first "automated zany map" on top of the block business?

Reply:
Automating something that we do not know how to do does not seem like the most ideal starting point.